### ISSUE TITLE
fix ruamel import problem with using Conda

### ISSUE THREAD

**Issue by ajtritt** [NeurodataWithoutBorders/pynwb#26]
Originally reported by: **Andrew Tritt (Bitbucket: [ajtritt](https://bitbucket.org/ajtritt), GitHub: [ajtritt](https://github.com/ajtritt))**

----------------------------------------

ruamel.yaml doesn't load properly when using conda. Loren had to change 'ruamel.yaml' to 'ruamel_yaml'

----------------------------------------
- Bitbucket: https://bitbucket.org/lblneuro/pynwb/issue/26

**Comment by ajtritt** [comment]
*Original comment by* **Loren Frank (Bitbucket: [lmfrnk](https://bitbucket.org/lmfrnk), GitHub: Unknown)**:

----------------------------------------

Just a note: I still have to change ruamel.yaml to ruamel_yamel in
src/form/spec/namespace.py

**Comment by ajtritt** [comment]
*Original comment by* **Andrew Tritt (Bitbucket: [ajtritt](https://bitbucket.org/ajtritt), GitHub: [ajtritt](https://github.com/ajtritt))**:

----------------------------------------

Loren, can you try uninstalling ruamel_yaml, and re-installing ruamel.yaml?

```
pip3 uninstall ruamel_yaml
pip3 install ruamel.yaml
```

**Comment by dorukozturk** [comment]
We published pynwb on conda-forge. I am still working on adding docs but here is the simplest way to get up and running on conda:
```
conda install -c conda-forge pynwb
```
The tests worked for me after I installed with this command on conda. 

On conda-forge ruamel.yaml is the correct name for that package (see [here](https://github.com/conda-forge/ruamel.yaml-feedstockl)). Also we are depending on that package in our feedstock which runs correctly (see [here](https://github.com/conda-forge/pynwb-feedstock/blob/master/recipe/meta.yaml#L29)).  

Let me know if the issue persists.
